### PR TITLE
[Testing] Params rebase

### PR DESCRIPTION
**Pull request by adriangb**

### CODE REVIEW

**Comment by adriangb** [comment]
@stsievert I rebased your branch in #22 onto the current master (I also wanted to test the new CI from a fork). Please feel free to do a reset on your `params` branch to this branch and I can delete this draft.

File: scikeras/wrappers.py [hunk 1]
```diff
@@ -137,17 +168,14 @@ def __init__(self, build_fn=None, **sk_params):
...
-        except AttributeError as e:
-            raise RuntimeError("Unasigned input parameter: {}".format(e))
+        params = self.get_params(deep=False)
+        for key in params.keys():
+            try:
+                getattr(self, key)
+            except AttributeError:
+                raise RuntimeError(
+                    "Unassigned input parameter: {}".format(key)
+                )
```
Comment (adriangb) [inline scikeras/wrappers.py:178]: This is unrelated to this PR, I'll open a seperate PR for it.

**Comment by stsievert** [comment]
So I should merge this branch into #22?

**Comment by adriangb** [comment]
That was the idea 👍

**Pull request closed without merging**